tests/test_sagemaker/test_sagemaker_endpoint.py

```diff
@@ -36,18 +48,18 @@ def sagemaker_client():
 
 
 @mock_sagemaker
-def test_create_endpoint_config(sagemaker_client):
+def create_endpoint_config(sagemaker_client, production_variants):
```

Comment:
Not sure whether we need the moto decorator for that helper function

Reply:
It shouldn't be necessary on the helper-function. From what I can see, the decorator are active on all the tests - so it will still be active when this method is invoked.
If you can just remove that, everything else looks good to me!

Reply:
Removed the decorator!